### PR TITLE
ENG-9951: AAUIW support for DROP STREAM

### PR DESCRIPTION
**Pull request by JadenW**

### CODE REVIEW

File: src/frontend/org/voltdb/compiler/DDLCompiler.java [hunk 1]
```diff
@@ -911,6 +922,22 @@ else if (matchStatus == ClassNameMatchStatus.NO_WILDCARD_MATCH) {
...
+        // matches if it is DROP STREAM
+        // group 1 is stream name
+        // guard against drop regular table
+        statementMatcher = SQLParser.matchDropStream(statement);
+        if (statementMatcher.matches()) {
+            String streamName = checkIdentifierStart(statementMatcher.group(1), statement);
+
+            if (isRegularTable(m_schema, streamName)) {
+                throw m_compiler.new VoltCompilerException(String.format(
+                        "Invalid DROP STREAM statement: table %s was not a stream.",
```
Comment (vtkstef) [inline src/frontend/org/voltdb/compiler/DDLCompiler.java]: wrong verb tense use `is` instead

File: tests/frontend/org/voltdb/compiler/TestDDLCompiler.java [hunk 2]
```diff
@@ -713,6 +713,86 @@ public void testCreateStreamNegative() throws Exception {
...
+            // compile successfully
+            boolean success = compiler.compileFromDDL(jarOut.getPath(), schemaPath);
+            assertTrue(success);
+
+            // cleanup after the test
+            jarOut.delete();
+        }
+    }
+
+    public void testDropStreamNegtive() throws VoltCompilerException {
```
Comment (vtkstef) [inline tests/frontend/org/voltdb/compiler/TestDDLCompiler.java]: spelling

File: tests/frontend/org/voltdb/compiler/TestDDLCompiler.java [hunk 3]
```diff
@@ -713,6 +713,86 @@ public void testCreateStreamNegative() throws Exception {
...
+        };
+
+        VoltCompiler compiler = new VoltCompiler();
+        for (int ii = 0; ii < schema.length; ++ii) {
+            File schemaFile = VoltProjectBuilder.writeStringToTempFile(schema[ii]);
+            String schemaPath = schemaFile.getPath();
+
+            // compile successfully
+            boolean success = compiler.compileFromDDL(jarOut.getPath(), schemaPath);
+            assertFalse(success);
```
Comment (vtkstef) [inline tests/frontend/org/voltdb/compiler/TestDDLCompiler.java]: other tests check the content of the failed message. please do so

Comment (JadenW) [inline tests/frontend/org/voltdb/compiler/TestDDLCompiler.java]: move the tests to testVoltCompiler. Also add check for the failed message.

File: src/frontend/org/voltdb/compiler/DDLCompiler.java (hunk 1 above)
Comment (JadenW) [inline src/frontend/org/voltdb/compiler/DDLCompiler.java]: fix, thanks

**Pull request merged**